### DISCUSSION TITLE
Media Queries not working

### DISCUSSION THREAD

**Discussion by zachrickards** [tailwindlabs/tailwindcss#9133] [Help] [answered]
Here is a set of code that isn't working due to !important on the flex class and not on the internal media query:

```
<div class="flex flex-wrap sm:flex-nowrap content-center">
   <div>text</div>
   <div>text 2</div>
</div>
```

This results in 

flex-wrap being !important and always overriding the media query of flex-nowrap

**Comment by adamwathan** [comment, marked as answer]
Generated CSS here is exactly what I'd expect, no `!important`:

https://play.tailwindcss.com/AzN0C2a8HI

<img width="753" alt="image" src="https://user-images.githubusercontent.com/4323180/185431927-33c1527c-fe16-4bb7-a90d-170ceddbbb80.png">

You'll have to provide a proper reproduction for anyone to be able to help with this.

**Reply by zachrickards** [reply]
100% apologize. This was due to my project conflicting with other things
